Dashboard/app/js/lib/views/maps/map-views-common.js

```diff
@@ -130,23 +123,14 @@ FLOW.NavMapsView = FLOW.View.extend({
         }
```

Comment:
Is the block above here `$.ajaxSetup()` still needed?

Reply:
Carto maps still uses ajax calls to get and create named maps so this block is still needed to display the loading icon when the calls are active